showcase/src/app/demo/pretty-print/demo-pretty-print.component.html

```diff
@@ -0,0 +1,58 @@
+<example-viewer [extensions]="[]"
```

Comment:
For this new "try it yourself" demo, you can use a simple card instead of the example viewer.

Later on, we will fix the styles for this type of cards to keep consistency